README.rst

```diff
@@ -17,13 +17,13 @@ Repository Structure
 
 * `/lib_xcore_math/ <https://github.com/xmos/lib_xcore_math/lib_xcore_math>`_ - The ``lib_xcore_math`` library directory.
 
-  * `api/ <https://github.com/xmos/lib_xcore_math/tree/master/lib_xcore_math/api/>`_ - Headers containing the public API for ``lib_xcore_math``.
-  * `script/ <https://github.com/xmos/lib_xcore_math/tree/master/lib_xcore_math/script/>`_ - Scripts used for source generation.
-  * `src/ <https://github.com/xmos/lib_xcore_math/tree/master/lib_xcore_math/src/>`_ - Library source code.
+  * `api/ <https://github.com/xmos/lib_xcore_math/tree/develop/lib_xcore_math/api/>`_ - Headers containing the public API for ``lib_xcore_math``.
```

Comment:
Did you mean to change the branch back from master to develop?

Reply:
yes, I did. I can undo the changes if you prefer master. I see pros and cons with either branches.

Reply:
There is a discussion about it in the one of the resolved comments

Reply:
Given the comments that @xross provided, I'm happy to point toward the develop branch.  I just wanted to make sure that the change was deliberate rather than a case of an unintended golbal search and replace.